### DISCUSSION TITLE
智能ABC输入习惯

### DISCUSSION THREAD

**Discussion by k0ngk0ng** [rime/home#1088] [General]
我是从智能ABC开始打中文的，智能ABC在输入编码时并不是实时显示候选字，直到按了空格才出现候选字。
我已经习惯这种按键方式很久，而且我的打字速度快慢也依赖这个习惯，虽然这会导致更多的按键次数
这个习惯除了QQ拼音可以设置，其他输入法好像没怎么见过（可能是我输入法用的少）

那么，如果想在rime还原这个习惯要怎么做？改代码还是改配置？我看了下配置好像没找到合适的。

谢谢

**Comment by LEOYoon-Tsaw** [comment]
試試朙月拼音．語句流

**Comment by lotem** [comment]
不支持的。
除了可以用数字做输入码，效率上似乎没有什么好处。
